rpcs3/Emu/RSX/rsx_utils.h

```diff
@@ -5,6 +5,8 @@ extern "C"
 #include <libavutil/pixfmt.h>
 }
 
+#include <cmath>
```

Comment:
I moved the include to stdafx.h.
As for the rebase, I clicked on the update branch button below, is that the same thing?